api/specs.proto

```diff
@@ -72,6 +72,9 @@ message ServiceSpec {
 	// UpdateConfig controls the rate and policy of updates.
 	UpdateConfig update = 6;
 
+	// Networks in ServiceSpec has been deprecated and is not used
+	// any more. This field is kept for compatibility only. It is
+	// replaced by Networks field in Task (TaskSpec).
```

Comment:
Can you please add that this field is still honored if it exists and `TaskSpec.Networks` does not?